pkg/yurtctl/util/edgenode/common.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package edgenode
 
 const (
-	KubeletSvcPath       = "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
+	KubeletSvcPath       = "/usr/lib/systemd/system/kubelet.service.d/10-kubeadm.conf"
```

Comment:
why is this updated?

Reply:
According to the issues mentioned by the community, KubeletSvcPath is generally "/usr/lib/systemd/system/kubelet.service.d/10-kubeadm.conf", so it was changed to the default value.

In addition, If the default value is "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf", the first time the user uses it, the conversion will often fail, and the parameter --kubeadm-conf-path must be passed in when converting. It is troublesome to use.

Reply:
can you share the issue address? 
want to know some background in detail

Reply:
issue: #572、#203 
May need further consideration, these two addresses are different in different systems.